DZ1_var11.py

```diff
@@ -0,0 +1,95 @@
+# Вариант 11
+# Коэффициенты системы линейных уравнений заданы в виде прямоугольной матрицы.
+# С помощью допустимых преобразований привести систему к треугольному виду.
+# Найти количество строк, среднее арифметическое элементов которых, меньше заданной величины.
+import random
+
+
+def get_rnd_matrix(min_val, max_val, row, col):
+    return [[random.randint(min_val, max_val) for _ in range(col)] for _ in range(row)]
+
+
+def validate_matrix(matrix):
+    col_max = len(matrix[0])
+
+    # перевірка даних
```

Comment:
write comments in English please.
if it's hard to you write using transliteration like:
# perevirka danyh